features/admin.feature

```diff
@@ -177,9 +177,6 @@ Feature:  Administration
       | domain     | status | canonical_domain |
       | search.gov | 200 OK |                  |
       | old.gov    | 200 OK | new.gov          |
-    And the following "searchgov urls" exist:
```

Comment:
The `Managing Search.gov Domains` scenario was getting rather unwieldy, so I've separated it into `Managing Search.gov Domains` and `Managing Search.gov URLs`. (Eventually I'd like to move each super admin page to its own feature file, but refactoring can wait.)